honeybot/plugins/translate.py

```diff
@@ -0,0 +1,340 @@
+# -*- coding: utf-8 -*-
+
+"""
+[weather.py]
+Translate Plugin
+[Author]
+Ahmed Deeb
+[About]
+A Google Translate plugin 
+[Commands]
+>>> . Google Translator object.
+
+    Examples
+    --------
+    translator = Plugin()
+
+    results1 = translator.translate("Einen schönen Tag allerseits")
+
+    # try 2 at a time
+    results2 = translator.translate(["Einen schönen Tag allerseits",
+                                     "Ich nehme an"])
+
+    # try detect
+    results3 = translator.detect("Einen schönen Tag allerseits")
+
+    # try to detect 2 at a time
+    results4 = translator.detect(["Einen schönen Tag allerseits",
+                                     "Ich nehme an"])
+   
+"""
+"""
+You need to fill in your API key from google below. Note that querying
+supported languages is not implemented.
+
+Language Code
+-------- ----
+Afrikaans 	af
+Albanian 	sq
+Arabic 	ar
+Belarusian 	be
+Bulgarian 	bg
+Catalan 	ca
+Chinese Simplified 	zh-CN
+Chinese Traditional 	zh-TW
+Croatian 	hr
+Czech 	cs
+Danish 	da
+Dutch 	nl
+English 	en
+Estonian 	et
+Filipino 	tl
+Finnish 	fi
+French 	fr
+Galician 	gl
+German 	de
+Greek 	el
+Hebrew 	iw
+Hindi 	hi
+Hungarian 	hu
+Icelandic 	is
+Indonesian 	id
+Irish 	ga
+Italian 	it
+Japanese 	ja
+Korean 	ko
+Latvian 	lv
+Lithuanian 	lt
+Macedonian 	mk
+Malay 	ms
+Maltese 	mt
+Norwegian 	no
+Persian 	fa
+Polish 	pl
+Portuguese 	pt
+Romanian 	ro
+Russian 	ru
+Serbian 	sr
+Slovak 	sk
+Slovenian 	sl
+Spanish 	es
+Swahili 	sw
+Swedish 	sv
+Thai 	th
+Turkish 	tr
+Ukrainian 	uk
+Vietnamese 	vi
+Welsh 	cy
+Yiddish 	yi
+"""
+import os
+import urlparse
+import urllib
+import urllib2
```

Comment:
woops this is a py3 bot, could you change please? if not within 3 days i'll merge it then change later

Reply:
I just made these changed the py imports now , is this how they should be?